docs/en/CSMM/advanced-feature-guide-fallingblocks.md

```diff
@@ -12,15 +12,17 @@ CPM's falling block prevention logs the event with some additional info in the f
 
 Naturally we will do this with a logLine hook, and we can use a simple search string `falling blocks prevented!` as our trigger. We need to add a couple of custom variables to this hook, but before we can do that we must save the hook. To do so, we first have to put a command in, as the hook cannot save with a blank command. For now, just enter `wait(1)` into the command field.
 
+Save the hook.
+
 Now that the hook is saved, we can edit it and create our variables. To create the variables, click the Edit button on the hook and look for the section "**Custom variables**". Our variables will be the X and Z coordinates (as we are not interested in checking altitude - the Y coordinate).
 
-For our first variable, enter the name `xcoord`. For this we need to identify the first 1 to 4 digit number following the @ in the log line, using the following regex:
+For our first variable, enter the name `xcoord`. use the following regex:
```

Comment:
Isn't it good that the regex gets explained a bit? I feel like this change will just delete information from the docs

Reply:
Yes, this is supposed to be an example for people to learn the whole process, not a script for people to add the hook without understanding what they actually did.

If the regex explanation is confusing at this point, perhaps it could be moved to a note after the regex?

Reply:
maybe add it back but make it clearer what it means because I have no idea

Reply:
```suggestion
For our first variable, enter the name `xcoord`. For this we need to identify the first 1 to 4 digit number following the @ in the log line, using the following regex:
```